ydb/core/tx/datashard/erase_rows_condition.cpp

```diff
@@ -104,20 +151,27 @@ class TExpirationCondition: public IEraseRowsCondition {
         default:
             Y_ABORT("Unreachable");
         }
-    }    
+    }
 
-    bool CheckStr(TStringBuf value) const {
-        switch (Type) {
-        // 'value since epoch' mode
-        case NScheme::NTypeIds::DyNumber:
-            if (const auto& wallClockDyNumber = GetWallClockDyNumber()) {
-                Y_ABORT_UNLESS(NDyNumber::IsValidDyNumber(value));
-                return value <= *wallClockDyNumber;
-            } else {
-                return false;
+    bool CheckSerialized(TStringBuf value) const {
+        if (const auto& wallClockDSerialized = GetWallClockSerialized()) {
```

Comment:
Имелось в виду:
```suggestion
        if (const auto& wallClockSerialized = GetWallClockSerialized()) {
```
?

Reply:
Поправил в https://github.com/ydb-platform/ydb/pull/10361